<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateEditDialog</class>
 <widget class="QDialog" name="DateEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date Edit Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QLabel" name="yearLabel">
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="monthLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="monthLabel">
     <property name="text">
      <string>Month:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="yearLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="dayLabel">
     <property name="text">
      <string>Day:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="dayLineEdit"/>
   </item>
  </layout>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>280</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
